### ISSUE TITLE
When adding cards view blacks out and all cards disappear

### ISSUE THREAD

**Issue by msauto** [zsarnett/Custom-Grid-View#3]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
2020.12
**Last working release (if known):**
na
**Browser and Operating System:**
chrome browser windows os

**Description of problem:**
Any time i add a card and sometimes when i edit a card the entire view gos black and all cards disappear. Only easy way i have found around this is to edit and add all cards before adding entry in raw config to engage drag n drop.

Any time i add a card and sometimes when i edit a card the entire view gos black and all cards disappear. Only easy way i have found around this is to edit and add all cards before adding entry in raw config to engage drag n drop.
Obviously it should not have this behavior.
Sorry forgot to get screenshot both times.
But i could foul it up on purpose and get one if needed.
**Javascript errors shown in the web inspector (if applicable):**

```

```

**Additional information:**
Nothing in logs when this happens.

**Comment by zsarnett** [comment]
That's not good. I'll look at this asap

**Comment by msauto** [comment]
Thanks Zac.
I'm not sure if this is helpful or not, but I added a new view with new cards and was able to edit as well as add a card but if I try to do so on any views that I have already built it blacks the screen and adds a "null" entry in several places on the raw config.

**Comment by zsarnett** [comment]
Fixed in 0.0.3

**Comment by msauto** [comment]
Excellent!

**Comment by msauto** [comment]
Sorry Zac but it doesn't seem to have fixed it. I pulled up this new view named tests2 which had 1 random card and i edited it just fine and even forgot to add the key back to the yaml after editing and was able to edit the card again and add the key back , all was fine until i added an alarm card. As soon as i clicked save in the card window the screen went black. I did try clearing cash as well with no joy.

![Screenshot (87)](https://user-images.githubusercontent.com/42117859/102157078-79608900-3e44-11eb-82d8-172951a66fdc.png)
![Screenshot (86)](https://user-images.githubusercontent.com/42117859/102157091-7cf41000-3e44-11eb-865f-c3d8d86a8286.png)
